hooks/useGas.ts

```diff
@@ -46,10 +66,35 @@ const useGas = () => {
 		}
 	}, []);
 
+	useEffect(() => {
+		const maxPriorityFeePerGas = selectedGas.maxPriorityFeePerGas;
+		const maxFeePerGasValue = isCustomGasPrice ? gasPriceWei : selectedGas.maxFeePerGas;
+
+		const l1GasConfig = { maxPriorityFeePerGas, maxFeePerGas: maxFeePerGasValue };
+		const l2GasConfig = { gasPrice: gasPriceWei };
+
+		const config = isL2 ? l2GasConfig : l1GasConfig;
```

Comment:
careful: in our case only mainnet (chainId = 1) requires the 1559 gas config. kovan, optimism mainnet, and optimism kovan all rely on the `gasPrice` field

Reply:
i updated this [one to `isMainnet`](https://github.com/Kwenta/kwenta/pull/377/files#diff-9b34fa921e4ee318e5ee18c3c3da462935b3f3a00188ce46bc41c66f5585d60aR32)